src/Drupal/MediaTrait.php

```
@@ -376,19 +377,20 @@ protected function mediaExpandEntityFieldsFixtures(\StdClass $stub): void {
     $fields = get_object_vars($stub);
 
     $driver = $this->getDriver();
-
-    if (!$driver instanceof DrupalDriver) {
-      throw new \RuntimeException('The current driver does not support Drupal-specific operations. Ensure you are using a compatible Drupal driver.');
+    if (!$driver instanceof DrupalDriverInterface) {
+      // @codeCoverageIgnoreStart
+      throw new \RuntimeException(sprintf('The active Drupal driver "%s" does not support content operations required for media field expansion.', $driver::class));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '382', column '17'). (undefined)

(MissingImport)